analyzers/src/SonarAnalyzer.Common/Rules/Hotspots/HardcodedIpAddressBase.cs

```diff
@@ -100,6 +115,18 @@ private void CheckForHardcodedIpAddressesInStringInterpolation(SyntaxNodeAnalysi
             && !IPAddress.IsLoopback(ipAddress)
             && !ipAddress.GetAddressBytes().All(x => x == 0); // Nonroutable 0.0.0.0 or 0::0
 
+        private bool IsInDocumentationBlock(IPAddress address) =>
+            address switch
```

Comment:
This should switch on address family, as that's the only switching condition. And return the matching condition instead of `true`

I have a feeling that the `Take.SequenceEquals` should be extracted (local static function?)